### DISCUSSION TITLE
Is it possible to select lines by a specific word?

### DISCUSSION THREAD

**Discussion by BrainDeLook** [SubtitleEdit/subtitleedit#7972] [Q&A]
Good afternoon! In Aegisub there is a function to select lines where a word is or is not contained with the possibility of selecting a column (Actor, subtitle, etc.) Is there such a possibility in subtitle edit?

**Comment by niksedk** [comment]
Somewhat. Try Edit - Modify selection:

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/2e128f92-daef-41fe-8160-4f148fa31e2a)

**Reply by BrainDeLook** [reply]
Thank you! It takes more steps than Aegisub, but it still works. Is there any functionality to invert line selection? That is, for example - I have selected a certain actor and I need to invert the selection to further remove unnecessary subtitles of other actors.

At the moment found only way: Select all lines - Remove the selection of a particular actor through "Modify selection"

**Reply by BrainDeLook** [reply]
Found the perfect way - Highlighting via "Actor" and it's very convenient! Thanks a lot for the tip)))
